tests/unit/conf/Config.js

```diff
@@ -89,3 +90,59 @@ describe('Site name', () => {
         assert.deepStrictEqual(newConfig.bootstrapList, expectedBootstrapList);
     });
 });
+
+describe('Config', () => {
+    describe('getReplicationSiteDestConfig', () => {
+        afterEach(() => {
+            delete process.env.BACKBEAT_CONFIG_FILE;
+            sinon.restore();
+        });
+        it('should return replication site destination config', () => {
+            process.env.BACKBEAT_CONFIG_FILE = `${__dirname}/configs/replicationMultiDestConfig.json`;
+            const conf = new Config();
+            const destConfig = conf.getReplicationSiteDestConfig('aws3');
+            assert.deepStrictEqual(destConfig, {
+                transport: 'https',
+                auth: {
+                    type: 'service',
+                    account: 'service-replication-3',
+                },
+                bootstrapList: [
+                    { site: 'aws1', type: 'aws_s3' },
+                    { site: 'aws2', type: 'aws_s3' },
+                    { site: 'aws3', type: 'aws_s3' }
+                ]
+            });
+        });
+
+        it('should return default replication destination config when site one is not available', () => {
+            process.env.BACKBEAT_CONFIG_FILE = `${__dirname}/configs/replicationMultiDestConfig.json`;
+            const conf = new Config();
+            sinon.stub(conf.extensions.replication, 'destination').value({
+                transport: 'https',
+                auth: {
+                    type: 'service',
+                    account: 'service-replication',
+                },
+                bootstrapList: [
+                    { site: 'aws1', type: 'aws_s3' },
+                    { site: 'aws2', type: 'aws_s3' },
+                    { site: 'aws3', type: 'aws_s3' }
```

Comment:
- no test for the validation of the joi config: may be good to have real _unit_ tests (i.e. just parsing a string with the `joi` template) to verify the various combinations, defaulting, ...
- no test for the `auth.vault` handling (_loadAdminCredentialsFromFile)